CHANGELOG.md

```diff
@@ -1,11 +1,22 @@
-4.55.0
+4.56.0
 ==========
 ### New Features
-- A user assigned managed identity can now be specified using its object ID. [4215](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/4215).  
+- MSAL.NET cache extensions ([Microsoft.Identity.Client.Extensions.Msal](https://www.nuget.org/packages/Microsoft.Identity.Client.Extensions.Msal)) package has been moved to the main MSAL.NET repository (where any [new issues](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues) should be created). The package version has been increased to match the main MSAL version. Along with this move, support for .NET 4.5 and .NET Core 3.1 was removed and this package now only supports .NET Standard 2.0. Additionally, [Microsoft.Identity.Client.Extensions.Adal](https://www.nuget.org/packages/Microsoft.Identity.Client.Extensions.Adal) has been deprecated. See [3152](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3152), [4330](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4330).
+- Added `AuthenticationResult.AuthenticationResultMetadata.Telemetry` that currently contains telemetry from the [Windows broker (WAM)](https://learn.microsoft.com/entra/msal/dotnet/acquiring-tokens/desktop-mobile/wam). See [4159](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4159).  
 
 ### Bug Fixes
+- Added throttling logic for acquiring tokens with managed identity to prevent the throttling exceptions thrown by the managed identity endpoints. See [4196](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4196).  
```

Comment:
```suggestion
- Added throttling logic for acquiring tokens for managed identity (using `AcquireTokenForManagedIdentity` and `WithAppTokenProvider`) to prevent the throttling exceptions thrown by the managed identity endpoints. See [4196](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4196).  
```